wradlib/comp.py

```diff
@@ -286,28 +291,45 @@ def transform_binned(sweep, raster):
         the transformation object
 
     """
-    wkt = raster.spatial_ref.attrs["crs_wkt"]
-    crs = pyproj.CRS.from_wkt(wkt)
-    sweep = sweep.wrl.georef.georeference(crs=crs)
-    coord_sweep = np.dstack((sweep.x, sweep.y))
-    x, y = np.meshgrid(raster.x, raster.y)
-    coord_raster = np.dstack((x, y))
+
+    try:
+        coord_sweep = np.dstack((sweep.x.values, sweep.y.values))
+    except AttributeError as err:
+        raise ValueError(
+            "Sweep has no x and y coordinates. Please georeference first."
+        ) from err
+
+    radar_crs = sweep.xradar.get_crs()
+    crs_wkt = raster.spatial_ref.attrs["crs_wkt"]
```

Comment:
Are we confident, that there will always be a `spatial_ref` variable? No blocker, but we should be prepared that users might hit an obstacle here.